github/resource_github_repository_test.go

```diff
@@ -83,6 +83,7 @@ func TestAccGithubRepository_basic(t *testing.T) {
 						DefaultBranch:       "master",
 						Archived:            false,
 					}),
+					testAccCheckGithubVulnerabilityAlerts(rn, false),
```

Comment:
in calls to `testAccCheckGithubVulnerabilityAlerts`, we'll want to send the repo name (in this case stored in the `repo` variable) as needed for the github API method; alternatively you could pass the whole `repo` object and extract the name inside the `testAccCheckGithubVulnerabilityAlerts` method